xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -255,12 +255,11 @@ Object.extend(XWiki, {
                         }
                       });
 
-                      // Replace the element's content with the temporary container's content, while also evaluating any inline scripts.
-                      // Note: This also does stripScripts internally, but it's OK now, since we have already added the necessary external scripts to head.
-                      $(extraID + "pane").update(container.innerHTML);
-
-                      // Notify the others that the DOM has been updated.
-                      document.fire('xwiki:dom:updated', {elements: [$(extraID + 'pane')]});
+                      require(['jquery'], function (jQuery) {
+                        jQuery("#" + extraID + "pane").html(container.innerHTML);
```

Comment:
not sure why you use jquery just for this line: sounds like the other line still use prototype.